frontend/src/layout/navigation-3000/sidebars/cohorts.ts

```diff
@@ -3,7 +3,7 @@ import { sceneLogic } from 'scenes/sceneLogic'
 import { Scene } from 'scenes/sceneTypes'
 import { urls } from 'scenes/urls'
 import { ExtendedListItem } from '../types'
-import type { cohortsSidebarLogicType } from './cohortsSidebarLogicType'
+import type { cohortsSidebarLogicType } from './cohortsType'
```

Comment:
I've assumed we have an implicit naming convention for logics to end with `..Logic.ts`, but just learned that the clean command works on `*Type.ts` and not `*LogicType.ts`. Guess my assumption doesn't hold?

Reply:
Yes, the convention definitely holds for logic const names, but the files don't necessarily need to be `Logic.ts`. Sometimes the logic is in the same file as a component, and I think in this case the `...SidebarLogic` pattern was a bit too wordy, as we're already in a `sidebars` directory.